regen-network/regen-web
# Feat: Create credit class UI + absolute paths, storybook in registry

## Description

Closes: regen-network/regen-registry#1012

This includes some changes from my recent tech debt exploration:
- allows storybook components to be built within the registry repo 
- moving `react/react-in-jsx-scope` to the -level config

Note: this PR breaks our storyshot setup (hence the large diff). I don't feel this is an issue personally, as we don't really review the snaps, and believe most agreed with that based on an architecture call, but if anyone objects to that, I can remove the UI work to its own PR while I try to fix -- the downside there is we couldn't use storybook in registry, and wouldn't have absolute imports

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. navigate to https://deploy-preview-1044--regen-registry.netlify.app/ecocredits/dashboard and authenticate with keplr
2. click the create credit class tab 
3. Fill out form, confirm UI in steps looks right

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
